mantatail.py

```diff
@@ -46,6 +74,11 @@ def __init__(self, port: int, motd_content: Optional[Dict[str, List[str]]]) -> N
         self.state = ServerState(motd_content)
 
     def run_server_forever(self) -> None:
+        """
+        Accepts incoming connections from clients.
+        Starts a receive loop on a separate thread,
+        listening for incoming commands from client.
+        """
```

Comment:
I don't like "listening" here, because in the context of sockets, it has a very specific meaning: being ready to accept new connections. That's not what the new threads do. How about something like this?

```suggestion
        """
        Accepts incoming connections from clients.
        Starts a separate thread to handle each connection.
        """
```